modules/Microsoft.Web/hostingEnvironments/.test/asev2/deploy.test.bicep

```diff
@@ -0,0 +1,85 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.web.hostingenvironments-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'whasev2'
+
+// =========== //
+// Deployments //
+// =========== //
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: location
+}
+
+module resourceGroupResources 'dependencies.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, location)}-paramNested'
+  params: {
+    networkSecurityGroupName: 'dep-<<namePrefix>>-nsg-${serviceShort}'
+    virtualNetworkName: 'dep-<<namePrefix>>-vnet-${serviceShort}'
+    subnetName: 'dep-<<namePrefix>>-subnet-${serviceShort}'
```

Comment:
I'd suggest to remove this parameter and just call the subnet (in the dependencies.bicep) 'defaultSubnet'. For the test you don't really care how the subnet inside the vnet is called.